mixnode/src/commands/describe.rs

```diff
@@ -20,7 +21,17 @@ pub fn command_args<'a, 'b>() -> App<'a, 'b> {
 
 pub fn execute(matches: &ArgMatches) {
     // figure out which node the user is describing
-    let id = matches.value_of("id").unwrap();
+    let id = matches
+        .value_of("id")
+        .expect("Please provide the id of your mixnode");
+
+    // ensure that the mixnode has in fact been initialized
```

Comment:
I think it would be better to attempt to load the config explicitly `Config::load_from_file(id)` so then you don't need to do any checks for directory existence etc. And also protects us from an edge case where user deleted the node content but preserved the directory